setup/symfony_server.rst

```diff
@@ -244,6 +244,72 @@ server provides a ``run`` command to wrap them as follows:
     # stop the web server (and all the associated commands) when you are finished
     $ symfony server:stop
 
+Docker Integration
+------------------
+
+The local Symfony server provides full `Docker`_ integration for projects that
+use it. First, make sure to expose the container ports:
+
+.. code-block:: yaml
+
+    # docker-compose.override.yaml
+    services:
+        database:
+            ports:
+                - "3600"
```

Comment:
Why does the port is 3600 and not 3306? MySQL is used commonly and probably it's better to user its port.

Reply:
Fixed! Thanks.